pkg/runtime/runtime.go

```diff
@@ -39,10 +41,20 @@ func NewRuntime(cfg *settings.Configuration, cmdArgs *parser.Arguments, version
 	logger := text.NewLogger(os.Stdout, os.Stderr, os.Stdin, cfg.Debug, "runtime")
 	runner := exe.NewOSRunner(logger.Child("runner"))
 
+	var transport = &http.Transport{}
```

Comment:
Can you copy the default transport? I think this works
```go
http.DefaultTransport.(*http.Transport).Clone() 
```



Can

Reply:
of course it is possible. I'm not an expert in Go at all :)